<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#CAE9FF</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>331</width>
     <height>431</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#e1f3ff</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Iniciar Sesión</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>139</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"> color: #1B4965</string>
     </property>
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_login_email">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>100</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="placeholderText">
      <string>Correo Electronico</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_login_password">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>140</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Contraseña</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#5FA8D3; color: white;</string>
     </property>
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Registro</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: 1B4965</string>
     </property>
     <property name="text">
      <string>Registrarse</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>261</width>
       <height>301</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#CAE9FF</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Dentista</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_register_dentist_name">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_dentist_lastname">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Apellidos</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_dentist_address">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Domicilio</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_dentist_license">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>140</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Cedula Profesional</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_dentist_email">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>170</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Correo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_dentist_password">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_register_dentist">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>230</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#5FA8D3; color: white;</string>
       </property>
       <property name="text">
        <string>Registrarse</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_register_dentist_age">
       <property name="geometry">
        <rect>
         <x>121</x>
         <y>80</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Edad:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Usuario</string>
      </attribute>
      <widget class="QLineEdit" name="lineEdit_register_user_name">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Nombre</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_user_lastname">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>50</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Apellidos</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_user_address">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Domicilio</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_user_email">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="placeholderText">
        <string>Correo</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_user_password">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>137</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contraseña</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_register_user">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#5FA8D3; color: white;</string>
       </property>
       <property name="text">
        <string>Registrarse</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_register_user_age">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>80</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:white;</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Edad:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
